gene/fromGtf.go

```diff
@@ -67,7 +75,7 @@ func GtfToGene(g *gtf.Gene, ref []*fasta.Fasta) *Gene {
 		answer.genomeSeq = make([]dna.Base, transcript.End-(transcript.Start-1))
 		copy(answer.genomeSeq, fastaMap[transcript.Chr][transcript.Start-1:transcript.End])
 		dna.ReverseComplement(answer.genomeSeq)
-		answer.cdnaSeq = make([]dna.Base, 0, gtf.CdsLength(transcript))
+		answer.cdnaSeq = make([]dna.Base, 0, len(answer.genomeSeq))
```

Comment:
Not a bug, but it seems like we are allocating more memory than needed.  Seems like this would be the max amount of memory needed (the case where there are no introns).

Reply:
Correct, this was just my quick & dirty way of adding space for indels.